qcodes/utils/path_helpers.py

```diff
@@ -0,0 +1,30 @@
+import os
+from pathlib import Path
+
+QCODES_USER_PATH_ENV = "QCODES_USER_PATH"
+
+
+def get_qcodes_path(*subfolder: str) -> str:
+    """
+    Return full file path of the QCoDeS module. Additional arguments will be
+    appended as subfolder.
+
+    """
+    import qcodes
```

Comment:
This does not belong in utils due to the import of qcodes.

Perhaps the best is to replace with using importlib-resoucrces 

https://importlib-resources.readthedocs.io/en/latest/#:~:text=importlib_resources%20is%20a%20library%20which%20provides%20for%20access,that%20is%20located%20within%20an%20importable%20Python%20package.